x-pack/plugins/security_solution/public/cases/components/configure_cases/mapping.test.tsx

```diff
@@ -32,186 +28,33 @@ describe('Mapping', () => {
   afterEach(() => {
     wrapper.unmount();
   });
-
   describe('Common', () => {
     test('it shows mapping form group', () => {
-      expect(wrapper.find('[data-test-subj="case-mapping-form-group"]').first().exists()).toBe(
-        true
-      );
+      expect(wrapper.find('[data-test-subj="static-mappings"]').first().exists()).toBe(true);
     });
 
-    test('it shows mapping form row', () => {
-      expect(wrapper.find('[data-test-subj="case-mapping-form-row"]').first().exists()).toBe(true);
+    test('correctly maps fields', () => {
+      expect(wrapper.find('[data-test-subj="field-mapping-source"] code').first().text()).toBe(
+        'title'
+      );
+      expect(wrapper.find('[data-test-subj="field-mapping-target"] code').first().text()).toBe(
+        'short_description'
+      );
     });
-
-    test('it shows the update button', () => {
+    // skipping until next PR
+    test.skip('it shows the update button', () => {
       expect(
-        wrapper.find('[data-test-subj="case-mapping-update-connector-button"]').first().exists()
+        wrapper.find('[data-test-subj="case-mappings-update-connector-button"]').first().exists()
       ).toBe(true);
     });
 
-    test('it shows the field mapping', () => {
-      expect(wrapper.find('[data-test-subj="case-mapping-field"]').first().exists()).toBe(true);
-    });
-
-    test('it updates thirdParty correctly', () => {
-      wrapper
-        .find('button[data-test-subj="case-configure-third-party-select-title"]')
-        .simulate('click');
-      wrapper.update();
-      wrapper.find('button[data-test-subj="dropdown-mapping-description"]').simulate('click');
-      wrapper.update();
-
-      expect(onChangeMapping).toHaveBeenCalledWith([
-        { source: 'title', target: 'description', actionType: 'overwrite' },
-        { source: 'description', target: 'not_mapped', actionType: 'append' },
-        { source: 'comments', target: 'comments', actionType: 'append' },
-      ]);
-    });
-
-    test('it updates actionType correctly', () => {
+    test.skip('it triggers update flyout', () => {
+      expect(setEditFlyoutVisibility).not.toHaveBeenCalled();
       wrapper
-        .find('button[data-test-subj="case-configure-action-type-select-title"]')
+        .find('button[data-test-subj="case-mappings-update-connector-button"]')
+        .first()
         .simulate('click');
-      wrapper.update();
-      wrapper.find('button[data-test-subj="edit-update-option-nothing"]').simulate('click');
```

Comment:
These tests will be re-added when the dynamic component is introduced in the next PR